MonoGame.Framework/Platform/Input/GamePad.iOS.cs

```diff
@@ -213,6 +215,11 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone leftDead
                     if (controller.Gamepad.ButtonY.IsPressed)
                         buttons |= Buttons.Y;
 
+                    if (controller.Gamepad.LeftShoulder.IsPressed)
```

Comment:
I'm surprised these didn't already exist, as the button appeared to be toggling correctly in @SimonDarksideJ 's GamePad app.

Reply:
I noticed a long time ago. I used to have a SteelSeries that was not extended that's when I noticed it was missing.

Reply:
Yeah, this is very curious, as testing worked for both MS and Sony GamePads on all platforms correctly.